.github/workflows/pull-request-test.yml

```
@@ -120,25 +120,20 @@ jobs:
       - name: Get AKS credentials
         if: needs.check-deploy.outputs.deploy == 'true'
         run: |
-          az aks get-credentials --resource-group renku-dev --name aks-switzerlandnorth-renku-dev --file "${{ github.workspace }}/renkubot-kube.config"
-          chmod 600 "${{ github.workspace }}/renkubot-kube.config"
-
-          echo "RENKUBOT_KUBECONFIG<<EOF" >> $GITHUB_ENV
-          cat "${{ github.workspace }}/renkubot-kube.config" >> $GITHUB_ENV
-          echo "EOF" >> $GITHUB_ENV
+          az aks get-credentials --resource-group renku-dev --name aks-switzerlandnorth-renku-dev --file "${{ runner.temp }}/renkubot-kube.config"
```

[we can do this later]
File and env var already setup by `azure/aks-set-context@v4`.
```suggestion
          az aks get-credentials --resource-group renku-dev --name aks-switzerlandnorth-renku-dev
```

---

Ok agreed. Lets just get this running.